### PR TITLE
Add list_length rule

### PR DESCRIPTION
**Pull request by Szymczak**
Added an extra list_length that validates if a list is of a certian size

### CODE REVIEW

File: src/main/java/com/szymczak/livr/Rules/CommonRules.java [hunk 1]
```diff
@@ -43,4 +44,32 @@
...
+        } else {
+            final Long length = Long.valueOf(object.toString());
+            return checkArrayLength(length, length);
+        }
+    };
+
+    private static Function<FunctionKeeper, Object> checkArrayLength(Long minLength, Long maxLength) {
+        return (FunctionKeeper wrapper) -> {
+            Object value = wrapper.getValue();
+            if (value == null || (value.toString()).equals("")) return "";
```
Comment (balachandarlinks) [inline src/main/java/com/szymczak/livr/Rules/CommonRules.java]:
```suggestion
            if (LIVRUtils.isNoValue(wrapper.getValue())) return "";
```
We can utilise this helper function here.

Comment (balachandarlinks) [inline src/main/java/com/szymczak/livr/Rules/CommonRules.java]: Is empty list valid? I would consider it as size 0 list and do the validation against the rule. What do you think?

**Review by balachandarlinks — CHANGES_REQUESTED** [review]

File: src/main/java/com/szymczak/livr/Rules/CommonRules.java (hunk 1 above)
Comment (Szymczak) [reply src/main/java/com/szymczak/livr/Rules/CommonRules.java]: Yes I think this would be valid if we wanted to check that the list is empty.

Comment (balachandarlinks) [reply src/main/java/com/szymczak/livr/Rules/CommonRules.java]: Got you. This case handles the object being empty or null 👍

Comment (Szymczak) [reply src/main/java/com/szymczak/livr/Rules/CommonRules.java]: 6592b7e

File: src/test/java/com/szymczak/livr/ListLengthTests.kt [hunk 2]
```diff
@@ -0,0 +1,195 @@
+package com.szymczak.livr
+
+import org.assertj.core.api.Assertions.assertThat
+import org.junit.Test
+import org.skyscreamer.jsonassert.JSONAssert
+
+class ListLengthTests {
```
Comment (balachandarlinks) [inline src/test/java/com/szymczak/livr/ListLengthTests.kt:7]: Good effort with the tests 👏

**Review by balachandarlinks — APPROVED** [review]

**Pull request merged**